README.md

```diff
@@ -83,6 +83,28 @@ You can set up our Nix module for GTK with the following config:
 
 Alternatively, if you are not using our Nix module, you can grab the theme from [nixpkgs/catppuccin-gtk](https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/themes/catppuccin-gtk/default.nix)
 
+## Flatpak
+> [!WARNING]
+> Flatpak package is not maintained by us. Due to that we cannot recommend using it as as there maybe some unexpected issues.
+
+Flatpak by default restricts access to themes, to allow access, use the following: 
+```bash
+sudo flatpak override --filesystem=$HOME/.local/share/themes
+```
+
+After you've allowed access, set the theme, using the following:
+```bash
+# Change to suite your flavor / accent combination
+export FLAVOR="mocha"
+export ACCENT="mauve"
+
+# Set the catppuccin theme directory
+export THEME_DIR="$HOME/.local/share/themes/catppuccin-${FLAVOR}-${ACCENT}-standard+default"
+
+# Set the theme
+sudo flatpak override --env=GTK_THEME=$THEME_DIR
```

Comment:
The env var is not a path but a name e.g. `GTK_THEME="catppuccin-mocha"`

This is also missing giving flatpak access to those dirs. `sudo flatpak override --filesystem=$HOME/.local/share/themes`

Reply:
An example of this might be `GTK_THEME="catppuccin-mocha-mauve-standard+default"` using the new standard for names